tests/e2e_tests/multistep/test_dendrite.py

```diff
@@ -126,11 +125,11 @@ async def test_dendrite(local_chain):
     # record logs of process
     # Create tasks to read stdout and stderr concurrently
     # ignore, dont await coroutine, just write logs to file
-    await asyncio.create_task(
+    asyncio.create_task(
         write_output_log_to_file("dendrite_stdout", dendrite_process.stdout)
     )
     # ignore, dont await coroutine, just write logs to file
-    await asyncio.create_task(
+    asyncio.create_task(
```

Comment:
These also need to be awaited. Same reason.

Reply:
According to Daniel, he uses this for debugging, so he asked to ignore it.